HTTP2-Core.package/Boolean.extension/instance/integerRepesentation.st

```diff
@@ -0,0 +1,3 @@
+*HTTP2-Core
+integerRepesentation
```

Comment:
Mir ist gerade aufgefallen, dass du integerRep_**r**_esentation falsch geschrieben hast.

Insgesamt wäre es mir allerdings lieber, wenn wir in der neuen Flags Klasse komplett ohne diese Extension auskommen würden.